components/nav/index.styl

```diff
@@ -6,7 +6,7 @@
   justify-content space-between
   align-items center
   border-bottom 1px solid rgba(white, 0.5)
-  z-index 501
+  z-index 505
```

Comment:
its also good practice to keep these kinds of values in one place (like a variables.styl file) so they are easy to keep track of. that way you change it in a single place if elements are showing up in the wrong order.